its/src/test/java/com/sonar/it/scanner/msbuild/utils/AnalysisContext.java

```diff
@@ -54,13 +57,27 @@ public static AnalysisContext forServer(String projectKey, Path temp, String dir
     return new AnalysisContext(ServerTests.ORCHESTRATOR, classifier, projectKey, TestUtils.projectDir(temp, directoryName), ServerTests.token());
   }
 
+  public static AnalysisContext forCloud(String projectKey, Path temp, String directoryName) {
+    var context = new AnalysisContext(null, ScannerClassifier.NET_FRAMEWORK, projectKey, TestUtils.projectDir(temp, directoryName), CloudConstants.SONARCLOUD_TOKEN);
+    context.begin
+      .setOrganization(CloudConstants.SONARCLOUD_ORGANIZATION)
+      .setProperty("sonar.scanner.sonarcloudUrl", CloudConstants.SONARCLOUD_URL)
+      .setProperty("sonar.scanner.apiBaseUrl", CloudConstants.SONARCLOUD_API_URL)
```

Comment:
The vargs approach could be used here as well.

Reply:
That would require to create the `new Property(..)` instances. So this multiline builder is better